src/main/java/seedu/address/model/person/Person.java

```diff
@@ -67,6 +68,15 @@ public Set<Role> getRoles() {
         return Collections.unmodifiableSet(roles);
     }
 
+    /**
+     * Check is Person is a member
+     * @return true is Person has a role "member"
+     */
+    public boolean isMember() {
+        Set<Role> roles = this.getRoles();
+        return roles.contains(new Role(Member.MEMBER_ROLE));
```

Comment:
Can consider using `new Member()` since `Member` is already extracted out from the `Role` as its subclass although `new Member()` is equivalent to `new Role(Member.MEMBER_ROLE)`